src/app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
             android:name=".ViewHabitEventActivity"
             android:exported="false" />
         <activity
-            android:name=".UserLoginActivity"
+            android:name=".ExampleActivity"
```

Comment:
Shouldn't we use `UserLoginActivity` as default?

Reply:
Just changed it for the purposes of testing this activity. Should I update it in this PR or just let people change it as they use it?

Reply:
Yup, I think we can just let people change it as they see fit!